cli/tests/activate.bats

```diff
@@ -2008,8 +2008,8 @@ EOF
   project_setup
   FLOX_SHELL="bash" run $FLOX_BIN activate --dir "$PROJECT_DIR" -- env
   assert_success
-  refute_output "FLOX_SHELL="
-  refute_output "_flox_shell="
+  refute_output --partial "FLOX_SHELL="
+  refute_output --partial "_flox_shell="
```

Comment:
Can you describe this change?

Reply:
Yeah .. I tripped across this bug from a previous commit and included it by accident - it could be a separate PR if you like. Without the `--partial` argument `refute_output` is checking to ensure the _entire_ output is the quoted string, so it was a mistake not to include that in the first place. I fixed this as I added further `refute_output --partial` invocations below.